census_view.cpp

```diff
@@ -1292,20 +1341,18 @@ void CensusView::update_visible_columns()
     // Reason: although the case and class defaults are hidden, they're
     // still information--so if the user made them different from any cell
     // wrt some column, we respect that conscious decision.
-    std::vector<std::string> const& all_headers(case_parms()[0].member_names());
-    int column = 0;
-    for(auto const& header : all_headers)
+    int column = 1;
```

Comment:
Why the change from initializing it to 0 and incrementing it in the beginning of the loop? IMHO the old/current version is more natural, initializing to 1 seems strange.